install/kubernetes/cilium/templates/cilium-configmap.yaml

```
@@ -882,6 +882,20 @@ data:
 {{- if .Values.hubble.redact.http }}
   # Enables redaction of the http URL query part in flows
   hubble-redact-http-urlquery: {{ .Values.hubble.redact.http.urlQuery | quote }}
+{{- if .Values.hubble.redact.http.headers }}
+  #  Enables redaction of http headers by specifying which headers will not be redacted
+{{- if .Values.hubble.redact.http.headers.allow }}
+  hubble-redact-http-headers-allow: {{- range .Values.hubble.redact.http.headers.allow }}
+    {{ . }}
```

I think we need to prepend with a dash in order for `range` to generate a proper list here 🤔 
```suggestion
    - {{ . }}
```

---

I believe this is intentional. The config option corresponds to a StringSlice flag, which takes a single string, with space separated values. We do this in a few other spots in the config like in the prometheus metrics list, etc.

---

Ah, I see! Disregard then.

---

Yeap it's intetional, it's the same with the `hubble-metrics` from a few lines above: https://github.com/cilium/cilium/blob/8c86d079949bee3c91567d9f98a02399a8193086/install/kubernetes/cilium/templates/cilium-configmap.yaml#L868-L872